code/modules/research/mechanic/flatpack.dm

```diff
@@ -147,6 +147,13 @@
 		if((stacked.len + stacking.stacked.len + 2) >= MAX_FLATPACK_STACKS) //how many flatpacks we can in a stack (including the bases)
 			to_chat(user, "You can't stack flatpacks that high.")
 			return
+		if(user.incapacitated() || user.lying)
```

Comment:
Well, this still seems to have the issue that it doesn't stop me stacking a flatpack from any distance if I'm next to the target. I don't like the copypaste - put it under a proc.